htdocs/core/tpl/admin_extrafields_add.tpl.php

```diff
@@ -137,10 +137,12 @@ function init_typeoffields(type)
 		});
 
 		/* Autofill the code with label */
+		<?php if (empty($conf->global->MAIN_EXTRAFIELDS_CODE_AUTOFILL_DISABLED)) : ?>
```

Comment:
Can you use instead
`if (!getDolGlobalString('MAIN_EXTRAFIELDS_CODE_AUTOFILL_DISABLED')) {`